sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/PropertiesUtils.java

```diff
@@ -167,12 +169,10 @@ public static Properties getFrontendPropertiesFromSpaConfigUrl(String url) {
 		return properties;
 	}
 
-	public static Properties extractFrontendProperties(InputStream inputStream) throws IOException {
+	public static Properties getFrontendPropertiesFromJson(String jsonData) throws JsonProcessingException {
 		Properties properties = new Properties();
-		String jsonData = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
```

Comment:
Why remove this from the above line and then duplicate it from wherever this method is to be called?

Reply:
I updated the PR.